rules/no-invalid-remove-event-listener.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+const methodCallSelector = require('./selectors/method-call-selector');
+
+const MESSAGE_ID = 'no-invalid-remove-event-listener';
+const messages = {
+	[MESSAGE_ID]:
+		'Store the result of this expression in a variable that is passed to both `removeEventListener` the corresponding `addEventListener`.'
+};
+
+const isBindCall = node =>
+	node.type === 'CallExpression' &&
+	node.callee.type === 'MemberExpression' &&
+	node.callee.property.name === 'bind';
```

Comment:
`node.callee.computed` should be tested `foo[bind]()` is not a bind call.

Reply:
wow, I did not expect those to share the same ast